### DISCUSSION TITLE
UNO platform can not build a simple application.

### DISCUSSION THREAD

**Discussion by AghaKhan** [unoplatform/uno#7270] [General]
Since I have updated Windows 10 nothing is working with UNO Platform. Kindly watch 2 videos uno.mp3 and uno2022.mp4. 
[Uno](https://1drv.ms/u/s!AtjIdnVOoJXtpNtqhfvOPEegggspuA?e=GK0P0y)
Best regards
Agha

**Comment by IsmailHassani** [comment]
As the error states, you're missing the .net sdk's.
Download and install it from here: [https://dotnet.microsoft.com/download/visual-studio-sdks](https://dotnet.microsoft.com/download/visual-studio-sdks)
Check also if you have installed all required workloads in Visual Studio.

**Reply by jeromelaban** [reply]
Indeed, this looks like a incorrect/broken Visual Studio installation, try reinstalling or repairing it. Also for VS 2022, you'll need to follow the steps here: https://platform.uno/docs/articles/get-started-vs-2022.html

**Comment by AghaKhan** [comment]
Something is wring with UNO, because it does not work with VS 2019 or VS 2022. I reinstalled 3 times but problem is same all the time. Kindly look the video. Also note I am unable to create any lib with Uno. [AllVideos](https://1drv.ms/u/s!AtjIdnVOoJXtpNtqhfvOPEegggspuA?e=V9x8Os) Please watch UnoAll.mp4

**Reply by jeromelaban** [reply]
This is not an error we can address as it's a generic .NET error, and this is likely to happen with other project types.

If you create a project with `dotnet new` does it load in visual studio?

**Comment by AghaKhan** [comment]
Hi jeromelaban:
Today I installed .NET 6, with Visual Studio 2022 current, but still I am unable to compile for Android or iPad.
Regards
Agh
![error](https://user-images.githubusercontent.com/5318363/141402676-121948a9-96b3-4e96-9df2-4f3143bf9985.jpg)
a

**Reply by IsmailHassani** [reply]
Did you also installed the corresponding workloads?

**Reply by jeromelaban** [reply]
VS 2022 has introduced breaking changes for .NET 6 iOS/Android that will be fixed soon in Uno.

**Comment by AghaKhan** [comment]
Hi Ismail Hassani:
Did you also installed the corresponding workloads?
what it means? 
I did install Uno template under .NET 6 and Windows 11. 
Please guide me if I am missing. 
I have written games with WPF, such a way where I am not using any WPF classes for images, but failed on Uno. Same code used to work under Uno, but I am not sure what happened, it stop working.

**Reply by Daoting** [reply]
I tried. Visual Studio 2022 current cannot load maui workloads. Only the preview version can load workloads. My current development environment is VS2019 (16.11.6) and VS2022 Preview (17.1.0 Preview 1.0). Everything's fine.

**Comment by AghaKhan** [comment]
Wow.
Let me check.
Thanks for update.
Best regards
Agha Khan

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows
